### DISCUSSION TITLE
3.5.0 sponsored installer still shows unsponsored

### DISCUSSION THREAD

**Discussion by cstadler333** [massCodeIO/massCode#340] [General]
I installed massCode 3.5.0 sponsored and it still shows as unsponsored. I'm not sure if something went wrong during the upgrade, or if it's not just me. Previously, I had 3.4.1-sponsored working perfectly.
I also already tried installing it again and then completely uninstalling massCode and installing fresh.
I'm on Windows 11 x64.

**Comment by antonreshetov** [comment]
@cstadler333 It's a public repo. Please open issue in private repo or mail me.
